style/visual.less

```diff
@@ -175,6 +175,204 @@
         margin-right: auto;
     }
 
+    .geoDiv{
```

Comment:
@samgehret would importing the css from mapbox-gl-geocoder using an import statement here? See how the gl-draw css works in this file for reference.

Reply:
@ryanbaumann Ah yes, I didn't see that part initially :)